src/main.cpp

```diff
@@ -139,7 +139,7 @@ class ElrsManager {
             try {
                 if (pingsSinceLastRx >= 2) {
                     fmt::print("No packets received after two pings, resetting device\n");
-                    crsfInterface.ResetDevice();
+                    //crsfInterface.ResetDevice();
```

Comment:
We want to retain resetting the device after enough pings, just increase the amount of pings after which we reset the device here instead. Perhaps something like 20, that would be 6 seconds?

Reply:
It seems that the `ResetDevice` in usb case do nothing. Put it back as it was. 
I also tried keeping the PingInterval at 1 second but increased the number of pings. This didn't help. It appears that the "ping speed" is much more important than the number of pings.

I'll take a break from researching this for now because all I'm doing is monkey patching, and it's starting to annoy me a bit. =)